docs/how-to-guides/themes/theme-json.md

```diff
@@ -1202,6 +1197,14 @@ The default value is 2em.
 }
 ```
 
+The setting for `blockGap` is a boolean, and its default value is `null`. This allows an extra level of control over style output. The `settings.spacing.blockGap` setting in a `theme.json` file accepts the following values:
```

Comment:
```suggestion
The setting for `blockGap` is either a boolean or `null` value and is `null` by default. This allows an extra level of control over style output. The `settings.spacing.blockGap` setting in a `theme.json` file accepts the following values:
```

Nit: A small wording change that might make the type of accepted values for `blockGap` clearer again.